src/main/java/org/openpnp/machine/reference/driver/GrblDriver.java

```diff
@@ -51,92 +49,84 @@
 /**
  * TODO: Consider adding some type of heartbeat to the firmware.  
  */
+
 public class GrblDriver extends AbstractSerialPortDriver implements Runnable {
+
```

Comment:
do you mean
+//	private static final double minimumRequiredVersion = 0.75;
+//	private double connectedVersion;

This are commented out. For now i prefer leaving it as is until
jogging is implemented, because when implementing
Jogging it can be possible that it is advisable to limit version numbering.
I don't have checked it for now.
On jogging, it can be advisable to limit only firmware builds that
have lookahead builded in
for not decellerating and stop during moves or direction/speed changes.